cms/jinja2/templates/components/streamfield/glossary_section_block.html

```diff
@@ -0,0 +1,27 @@
+{% from "components/accordion/_macro.njk" import onsAccordion %}
+
+<section id="{{ value.title|slugify() }}">
+
+    <h2 class="ons-u-fs-l">{{ value.title }}</h2>
+
+    {% set glossary_terms=[] %}
+
+    {% for glossary_term in value.content %}
+        {% do glossary_terms.append({
+        "title": glossary_term.title,
+        "content": glossary_term.definition|richtext
+        })
+        %}
+    {% endfor %}
+
+    {# fmt:off #}
+    {{
+        onsAccordion({
+            "id": "glossary-terms-accordion",
+            "itemsList": glossary_terms
+        })
```

Comment:
```suggestion
            "allButton": {
                "open": _("Show all"),
                "close": _("Hide all")
            }
        })
```

https://nwp-prototype.ons.gov.uk/economy/inflation-and-price-indices/consumer-price-inflation/december-2024/#definitions